taskcluster-hook-pipeline-start.json

```diff
@@ -0,0 +1,51 @@
+{
+  "created": {
+    "$fromNow": "0 seconds"
+  },
+  "deadline": {
+    "$fromNow": "2 hours"
+  },
+  "expires": {
+    "$fromNow": "1 month"
+  },
+  "extra": {},
+  "metadata": {
+    "description": "",
+    "name": "BugBug data pipeline",
+    "owner": "babadie@mozilla.com",
+    "source": "https://github.com/mozilla/bugbug"
+  },
+  "payload": {
+    "artifacts": {
+    },
+    "cache": {
+    },
+    "capabilities": {},
+    "command": [
+      "/hello",
+    ],
+    "env": {
+    },
+    "features": {
+      "taskclusterProxy": true
+    },
+    "image": "hello-world",
+    "maxRunTime": 7200
+  },
+  "priority": "normal",
+  "provisionerId": "aws-provisioner-v1",
+  "retries": 5,
+  "routes": [
+    "index.project.relman.services.project.bugbug.data_pipeline.latest"
+  ],
+  "schedulerId": "-",
+  "scopes": [
+    "secrets:get:project/relman/bugbug/deploy",
```

Comment:
The needed secret will be project/relman/bugbug/production (project/relman/bugbug/deploy is for deployments, project/relman/bugbug/production is for operation)